wordpress-mobile/WordPress-iOS
# Corrected inadvertently modified localized string

### Description

Fixes #10721.

To test:

1. Checkout the branch and confirm that existing tests pass.
1. Visually inspect that both [this file](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Resources/en.lproj/Localizable.strings#L1172-L1173) and [this one](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/NUX/SiteCreationSiteDetailsViewController.swift#L85) look correct.
1. Follow the steps in the reported issue, and observe that the screen once again behaves as expected. See the title text below.

<img width="584" alt="corrected" src="https://user-images.githubusercontent.com/221062/50376406-4cfd3380-05c1-11e9-842d-2a78567a1cf0.png">

Update release notes:
